media/css/firefox/all.less

```diff
@@ -276,6 +276,7 @@ body.nightly {
     }
 
     #language-search {
+        width: @widthTablet - (@gridGutterWidth * 2 * 2.25);
```

Comment:
Rather than make this smaller to fit it's container I think it might be better to make `.search-column` larger at this breakpoint, to make better use of available space. It could probably remain the same size as it was at desktop width.